### PR TITLE
Add iex> to make some doctests run

### PR DESCRIPTION
**Pull request by wojtekmach**

### CODE REVIEW

File: lib/elixir/lib/code.ex [hunk 1]
```diff
@@ -50,9 +50,11 @@ defmodule Code do
 
   ## Examples
 
-      Code.append_path(".") #=> true
+      iex> Code.append_path(".")
```
Comment (wojtekmach) [inline lib/elixir/lib/code.ex]: this line (and similar) has side effects so I'm not 100% sure if it won't mess with other tests

Comment (lexmag) [inline lib/elixir/lib/code.ex]: Many of `Code` examples don't use `iex>` intentionally, because of undesirable side-effects.

Comment (wojtekmach) [inline lib/elixir/lib/code.ex]: right. I'll just revert changes in code.ex except for `available_compiler_options` which is pure

**Comment by josevalim** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Pull request merged**